app/assess/routes.py

```diff
@@ -294,8 +295,17 @@ def landing():
     of applications and their statuses
     """
 
+    search_params = {
+        "search_term": "",
+    }
```

Comment:
Do we want to pass this through as empty if not specified? Can we make it `None` in that case?

Reply:
No as it will add None to the search bar
![image](https://user-images.githubusercontent.com/97108643/202135952-5b9089ea-19b2-4876-a9a6-91188d7cbadf.png)

Reply:
@adamdavies1 I think we could solve this by adding a condition to the template e.g.

`{{ query.search_term if query.search_term }}`